en/debug/probe_bmp.md

```diff
@@ -0,0 +1,66 @@
+# Black Magic Probe (and Dronecode Probe)
```

Comment:
Note, added both names - I want this to show up in searches and sidebar so if people know about the name they see the relationship.